mapping.md

```diff
@@ -57,3 +57,18 @@
 | `type`         | `-`            | `type`              |
 | `uniq`         | `uniq`         | `uniq`              |
 | `uniqBy`       | `uniqBy`       | `uniqBy`            |
+
+### Helpful one-liners
+
+Some lodash and ramda functions don't have a Remeda equivalent, but can be
+easily replaced with a one-liner in TypeScript. Some of the most common
+are listed below.
+
+| Lodash         | Ramda          | TypeScript                                 |
+| -------------- | -------------- | ------------------------------------------ |
+| `add`          | `add`          | `(x, y) => x + y`                          |
+| `isNil`        | `isNil`        | `x => x == null`                           |
+| `nth`          | `nth`          | `n => a[n]`                                |
+| `-`            | `path`         | `a?.b?.c`                                  |
+| `-`            | `propEq`       | `x => x[a] === value`                      |
+| `-`            | `propOr`       | `x => x[a] ?? defaultValue`                |
```

Comment:
Could you change `x => x[a]` to `x => x.a`?

Reply:
Sure